CMakeLists.txt

```diff
@@ -48,6 +48,11 @@ include(CheckIncludeFiles)
 include(CheckIncludeFileCXX)
 include(CheckFunctionExists)
 
+#put all the libs and binaries in one place
+set(CMAKE_ARCHIVE_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/lib)
```

Comment:
Was this actually necessary?  I don't necessarily have a problem with introducing lib, bin folders, but it seems unrelated to this change.

Presumably vstart will need updating, as will https://github.com/ceph/ceph-qa-suite/blob/master/tasks/cephfs/vstart_runner.py

Might it make sense to do the lib/ bin/ thing in another patch later?